src/app/destiny2/d2-definitions.ts

```diff
@@ -179,6 +179,9 @@ export function buildDefinitionsFromManifest(db: AllDestinyManifestComponents) {
         if (!dbTable) {
           throw new Error(`Table ${table} does not exist in the manifest`);
         }
+        if (id < 1 || id > Number.MAX_SAFE_INTEGER) {
```

Comment:
Don't we want the inverse of these conditions, i.e. `id > 1 && id < Number.MAX_SAFE_INTEGER`?

Reply:
no because this throws an error if the has is out of range

Reply:
Ah got it, I missed it was all new and was just thinking of excluding everything outside.